x/lockup/keeper/grpc_query_test.go

```diff
@@ -513,3 +513,10 @@ func (suite *KeeperTestSuite) TestLockedDenom() {
 	testTotalLockedDuration("2h", 0)
 	testTotalLockedDuration("1h", 10)
 }
+
+func (suite *KeeperTestSuite) TestParams() {
```

Comment:
Can we add test cases for setting params and then querying it?

Reply:
Yeah sure